docker/Dockerfile.olsolr

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:xenial
+
+RUN apt-get -qq update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y solr-tomcat && \
+    ln -s /var/log/tomcat7/ /usr/share/tomcat7/logs && \
+    ln -s /etc/tomcat7/ /usr/share/tomcat7/conf && \
+    sed -i "s/8080/8983/g" /etc/tomcat7/server.xml
```

Comment:
I wonder if we need to do this port substitution now it is it's in its own container, there should be nothing to conflict with, and we can map the actual ports however we like between containers.